bin/node/runtime/src/lib.rs

```diff
@@ -199,7 +198,7 @@ parameter_types! {
 const_assert!(NORMAL_DISPATCH_RATIO.deconstruct() >= AVERAGE_ON_INITIALIZE_RATIO.deconstruct());
 
 impl frame_system::Config for Runtime {
-	type BaseCallFilter = ();
+	type BaseCallFilter = Everything;
```

Comment:
As far I understand it it is a refactoring. Now we need to explicitly say that all calls to the runtime from SC are allowed